controllers/iperf3/serverdeployment.go

```diff
@@ -49,9 +48,14 @@ func (r *Reconciler) newServerDeployment(ctx context.Context, cr *perfv1alpha1.I
 		labels[k] = v
 	}
 
-	iperfCmdLineArgs := fmt.Sprintf("-s -p %s %s",
-		strconv.Itoa(iperf3ServerPort),
-		cr.Spec.ClientConfiguration.CmdLineArgs)
+	iperfCmdLineArgs := fmt.Sprintf("--server --port %s ",
```

Comment:
I prefer string arrays and `strings.Join` instead of concatenating strings. In this case you do not have to handle spaces at the end. Also `%d` formats integers.

```golang
var cmds []string = []string{fmt.Sprintf("--server --port %d", iperf3ServerPort)}
if cr.Spec.UDP {
    cmds = append(cmds, "--udp")
}
cmds = append(cmds, cr.Spec.ClientConfiguration.CmdLineArgs)
```

Reply:
done, with some modification to the suggestion. now we're handling quoteing using qsplit pkg